content/guides/hot-module-replacement.md

```diff
@@ -18,125 +19,194 @@ related:
     url: /api/hot-module-replacement
 ---
 
+T> This guide extends on code examples found in the [`Development`](/guides/development) guide.
+
 Hot Module Replacement (or HMR) is one of the most useful features offered by webpack. It allows all kinds of modules to be updated at runtime without the need for a full refresh. This page focuses on __implementation__ while the [concepts page](/concepts/hot-module-replacement) gives more details on how it works and why it's useful.
 
 W> __HMR__ is not intended for use in production, meaning it should only be used in development. See the [building for production guide](/guides/production) for more information.
 
-
 ## Enabling HMR
 
-This feature is great for productivity. Let's take a look at how to set it up with [webpack-dev-server](https://github.com/webpack/webpack-dev-server)...
-
-``` js
-const path = require('path');
-const webpack = require('webpack');
-
-module.exports = {
-  entry: './index.js',
+This feature is great for productivity. All we need to do is update our [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration, and use webpack's built in HMR plugin.
 
-  plugins: [
-    new webpack.HotModuleReplacementPlugin() // Enable HMR
+``` diff
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
++ const webpack = require('webpack');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
+    devServer: {
+      contentBase: './dist',
++     hot: true
+    },
+    plugins: [
+      new HtmlWebpackPlugin({
+        title: 'Hot Module Replacement'
+      }),
++     new webpack.HotModuleReplacementPlugin()
   ],
-
   output: {
-    filename: 'main.js',
-    path: path.resolve(__dirname, 'dist'),
-    publicPath: '/'
-  },
-
-  devServer: {
-    hot: true, // Tell the dev-server we're using HMR
-    contentBase: path.resolve(__dirname, 'dist'),
-    publicPath: '/'
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist')
   }
 };
 ```
 
-Not too bad, huh? Let's test it out using `module.hot.accept`...
+You can also use the CLI to modify the [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration with the following command: `webpack-dev-server --hotOnly`.
+
+Now let's update the `index.js` file so that when a change inside `print.js` is detected we tell webpack to accept the updated module.
```

Comment:
I think you need to mention that the reader should `npm start` their `webpack-dev-server` before altering the `print.js` module.